### PR TITLE
Set "Trying to use another call cache hit" messages to debug [29 Hotfix]

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

**Comment by geoffjentry** [comment]
👍

File: engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala [hunk 1]
```diff
@@ -515,11 +515,11 @@ class EngineJobExecutionActor(replyTo: ActorRef,
 
     data.ejha match {
       case Some(ejha) =>
-        log.info("Trying to use another cache hit for job: {}", jobDescriptorKey)
+        workflowLogger.debug("Trying to use another cache hit for job: {}", jobDescriptorKey)
```
Comment (mcovarr) [inline engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala]: is the stringification of the `jobDescriptorKey` okay?

Comment (cjllanwarne) [reply engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala]: it's not *great* but it's surprisingly readable

Comment (cjllanwarne) [reply engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala]: Example:
```
Trying to use another cache hit for job: BackendJobDescriptorKey_TaskCallNode_foo.bar:20:1
```

Comment (mcovarr) [reply engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala]: I can has workflow id?

Comment (cjllanwarne) [reply engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala]: Yeah, that's why I also switched it to workflowLogger

Comment (mcovarr) [reply engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala]: sweet

**Pull request merged**